src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +186,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return Val.wrap(orientation.widthEstimateProperty(this));
+    }
+
+    Val<Double> totalHeightEstimateProperty() {
+        return Val.wrap(orientation.heightEstimateProperty(this));
```

Comment:
The about comment about avoiding wrapping by changing the return value of `orientation.heightEstimateProperty()` also applies here.